string base_status
string arm_status
string gripper_status
